### PR TITLE
add For Map Visualization - Bahrain is not available #5839

### PR DESCRIPTION
**Pull request by gaecoli**
## What type of PR is this? 

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [ ] Manually
- [ ] N/A

### CODE REVIEW

**Comment by gaecoli** [comment]
Bahrain, the GCC Gulf country, was not defined in the previous geo. Now I downloaded their country’s json code from the Internet and am now filling it in. Because the country design has many problems, I have not done any new addition or deletion operations of my own.

**Comment by gaecoli** [comment]
image

**Comment by gaecoli** [comment]
I don't know if the exact functions of the state should be merged or not, it might be more appropriate for you to decide. @justinclift

**Comment by justinclift** [comment]
I have no objection to including Bahrain.  From a quick look over Wikipedia, there don't seem to be any major border disputes either.

> Now I downloaded their country’s json code from the Internet ...

Where did you download it from?  We will need to make sure it was from a place that won't have a problem (eg licensing) with us using that.

**Comment by gaecoli** [comment]
@justinclift https://cartographyvectors.com/map/857-bahrain-detailed-boundary

**Comment by gaecoli** [comment]
> @justinclift https://cartographyvectors.com/map/857-bahrain-detailed-boundary

This use google map api.

**Comment by justinclift** [comment]
That particular page says:

image

We'll need to figure out what kind of "Attribution" they need, then if it's something we're ok with, we can include that with this PR.

**Comment by gaecoli** [comment]
> That particular page says:
> 
> image
> 
> We'll need to figure out what kind of "Attribution" they need, then if it's something we're ok with, we can include that with this PR.

What this?

**Comment by justinclift** [comment]
It's from the top right part of the page:

image

When a place says they want "attribution", it generally means they want you to tell people where you got the data from.  For example:

"We use the Bahrain border data from https://cartographyvectors.com"

That website doesn't seem to have any information about *where* they want that attribution notice, so we can probably just put it in a text file in our source code repo.

**Comment by gaecoli** [comment]
> It's from the top right part of the page:
> 
> image
> 
> When a place says they want "attribution", it generally means they want you to tell people where you got the data from. For example:
> 
> "We use the Bahrain border data from https://cartographyvectors.com"
> 
> That website doesn't seem to have any information about _where_ they want that attribution notice. So we can probably just put it in a text file in our source code repo.

Ok, i understood you.

**Comment by justinclift** [comment]
For now, lets create a new text file named "LICENSE.borders" and have its contents be:

```
The Bahrain map data used in Redash was downloaded from
https://cartographyvectors.com/map/857-bahrain-detailed-boundary in PR #6192.
* Free for personal and commercial purpose with attribution.
```

That should be good enough, and we can add more attribution notices there if we need to download more map data later on.

**Comment by gaecoli** [comment]
I added than file.

File: LICENSE.borders [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The Bahrain map data used in Redash was downloaded from
+
```
Comment (justinclift) [inline LICENSE.borders:2]: This line break in the middle of the text looks weird. :wink:

**Comment by gaecoli** [comment]
done!

**Review by justinclift — APPROVED** [review]
Looks good to me. :smile:

**Pull request merged**